modules/slo/variables.tf

```diff
@@ -1,26 +1,5 @@
 variable "datadog_slos" {
-  type = map(object({
-    name = string
-    type = string
-
-    thresholds = list(object({
-      target    = string
-      timeframe = string
-      warning   = string
-    }))
-
-    query = map(any)
-
-    groups      = list(string)
-    monitor_ids = list(number)
-
-    message      = string
-    description  = string
-    force_delete = bool
-    validate     = bool
-
-    tags = list(string)
-  }))
+  type        = any
```

Comment:
Is structure too compicated to define it as type?

Reply:
The problem is not the structure, its that we want optional variables. we want enabled to be optional with a default of true. but if we add it to the struct then you have to explicitly say on every SLO that `enabled: true` which becomes a pain